pkgs/development/tools/rust/cargo-pgx/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromGitHub, rustPlatform, pkg-config, openssl, libiconv, Security }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-pgx";
+  version = "0.4.5";
+
+  src = fetchFromGitHub {
+    owner = "tcdi";
+    repo = "pgx";
+    rev = "v${version}";
+    sha256 = "sha256-M81n4ORhMYpXRonJxEgFMOuR0Deb5CN2df70ltviIqI=";
+  };
+
+  cargoSha256 = "sha256-8x2oTnFWDx+57OKamci78WRPh+dVBj6SKJALlOBHuSg=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ openssl ]
+    ++ lib.optionals stdenv.isDarwin [ libiconv Security ];
+
+  buildAndTestSubdir = "cargo-pgx";
+
+  meta = with lib; {
+    description = "Cargo subcommand for ‘pgx’ to make Postgres extension development easy";
+    homepage = "https://github.com/tcdi/pgx/cargo-pgx";
```

Comment:
That's not the correct URL (404s), maybe it'd be best to link to the [main `pgx` project page](https://github.com/tcdi/pgx)? Or you could link to the correct URL, https://github.com/tcdi/pgx/tree/master/cargo-pgx. Your call.

Reply:
I would link to the versioned version of that page.

Reply:
done